mesa/experimental/cell_space/cell.py

```diff
@@ -123,30 +132,34 @@ def __repr__(self):  # noqa
 
     # FIXME: Revisit caching strategy on methods
     @cache  # noqa: B019
-    def neighborhood(self, radius=1, include_center=False):
+    def neighborhood(self, radius: int = 1, include_center: bool = False):
```

Comment:
would it be possible to add a docstring. 

Also, and not perse necessary for this PR, but the name of this method has been tripping me up. The name suggests that it is an attribute, so perhaps at some point we should change this to something like `get_neighborhood`

Reply:
agreed, there is some revisiting to be done here. I think at one point the idea was that this basically is an attribute/property for the default case, which is just a CellCollection of the the connections. Maybe we could actually have something like 

```python
def get_neighborhood(self, radius: int = 1, include_center: bool = False):
     ...

@property
def neighborhood(self):
    return self.get_neighborhood()
``` 

So for the "normal" neighborhood you could do `cell.neighborhood` and for everything more complex `cell.get_neighborhood(...)`

Reply:
Yes, this makes good sense. The most common case is indeed getting the direct neighborhood and it would be good if that is just a property. Naming-wise, what about `direct_neighborhood` and `get_neighborhood`? It would be nice if the names were a bit more distinctive.

Reply:
Yes more distinctive names sounds good. I'll have to think about `direct_neighborhood` cause I think its a bit verbose. But lets leave that and adding a docstring to a different PR